lib/_http_incoming.js

```diff
@@ -80,6 +82,9 @@ function IncomingMessage(socket) {
   // Flag for when we decide that this message cannot possibly be
   // read by the user, so there's no point continuing to handle it.
   this._dumped = false;
+
+  // Catch errors to avoid crashes now that autoDestroy is enabled
+  this.on('error', noop);
```

Comment:
Regarding https://github.com/nodejs/node/pull/33035#issuecomment-641124262 I went for option 1

Reply:
This causes this test to fail though:

![image](https://user-images.githubusercontent.com/15520377/88365940-e1281180-cd87-11ea-9434-402fbd9248f2.png)

Is this acceptable? If not, I think I should write a custom `destroy` implementation because the standard one *always* emits an `error` event in case of errors.

Reply:
By acceptable I mean that this behavior is acceptable and we can remove/refactor this test case.

Reply:
Sorry but this is a nono. We can't swallow errors like that.

Reply:
Understood. I'll work on a destroy method that implements the logic that was on _http_server.js. Thanks.